### PR TITLE
Revert unneeded kvutils format version bump

### PR DESCRIPTION
**Pull request by fabiotudone-da**
CHANGELOG_BEGIN
CHANGELOG_END

This format version bump turned out not to be needed (b/c the new format is backwards-compatible) and to cause dump compatibility test failures.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by fabiotudone-da** [comment]
@moritzkiefer-da FYI

**Comment by fabiotudone-da** [comment]
Yes; my assessment is that the snapshot that incorporates it breaks KV data continuity with previous snapshots and releases (that is going to be restored in the snapshot that incorporates this PR). @miklos-da Do you agree?

**Comment by cocreature** [comment]
kvutils doesn’t provide data continuity anyway and we broke this before so this seems harmless

**Comment by miklos-da** [comment]
> Yes; my assessment is that the snapshot that incorporates it breaks KV data continuity with previous snapshots and releases (that is going to be restored in the snapshot that incorporates this PR). @miklos-da Do you agree?

Yes, agreed, upping the version in the last snapshot was not necessary. I'm fine with reverting the version bump, though, keeping a note of the change would be useful.

**Comment by fabiotudone-da** [comment]
> > Yes; my assessment is that the snapshot that incorporates it breaks KV data continuity with previous snapshots and releases (that is going to be restored in the snapshot that incorporates this PR). @miklos-da Do you agree?
> 
> Yes, agreed, upping the version in the last snapshot was not necessary. I'm fine with reverting the version bump, though, keeping a note of the change would be useful.

Can you suggest a change in the version history comment for that?

**Comment by miklos-da** [comment]
> > > Yes; my assessment is that the snapshot that incorporates it breaks KV data continuity with previous snapshots and releases (that is going to be restored in the snapshot that incorporates this PR). @miklos-da Do you agree?
> > 
> > 
> > Yes, agreed, upping the version in the last snapshot was not necessary. I'm fine with reverting the version bump, though, keeping a note of the change would be useful.
> 
> Can you suggest a change in the version history comment for that?

Possibly move the change to the previous version (5)? I.e., it has been done as part of that version after this revert.

File: ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Version.scala [hunk 1]
```diff
@@ -83,8 +83,7 @@ object Version {
     *   5: * Add active_at to DamlContractKeyState to be able to check causal monotonicity of positive key lookups,
     *        i.e. whether the contract currently associated with a contract key was created in a transaction with
     *        ledger_effective_time <= the ledger_effective_time of the transaction under validation.
-    *   6: * Remove fields record_time from DamlCommandDedupValue and DamlSubmissionDedupValue.
     *
```
Comment (fabiotudone-da) [inline ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Version.scala:86]:
```suggestion
    *      * Remove fields record_time from DamlCommandDedupValue and DamlSubmissionDedupValue.
    *
```

**Comment by fabiotudone-da** [comment]
> Possibly move the change to the previous version (5)? I.e., it has been done as part of that version after this revert.

Done.

**Pull request merged**